tools/azsdk-cli/Azure.Sdk.Tools.Cli/Configuration/RequirementsV1.json

```diff
@@ -0,0 +1,121 @@
+{
+    "categories": {
```

Comment:
@weidongxu-microsoft @haolingdong-msft @msyyc @tadelesh @MaryGao 
can you please review the list and call out if there're any tools missing which are needed for mgmt-plane SDK?